src/components/reusuables/sections/TeamSection.js

```diff
@@ -0,0 +1,88 @@
+import React from "react"
+import { Container, Row, Col } from "react-bootstrap"
+import TeamCard from "../cards/TeamCard"
+import YohDevButton from "../YohDevButton"
+import ClientLogosInline from "../content/ClientLogosInline"
+
+const TeamSection = (props) => {
```

Comment:
Destructure props like the last review.
(props) should be ({heading, subtitle, etc})